app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -402,21 +401,4 @@ private void startScreenService() {
                 Context.MEDIA_PROJECTION_SERVICE);
         startActivityForResult(projectionManager.createScreenCaptureIntent(), SCREEN_CAPTURE_REQ_CODE);
     }
-
-    private String getRealPathFromUri(Context context, Uri contentUri) {
-        Cursor cursor = null;
-        try {
-            String[] proj = {MediaStore.Images.Media.DATA};
-            cursor = context.getContentResolver().query(contentUri, proj, null, null,
-                    MediaStore.Images.ImageColumns.DATE_TAKEN + " DESC");
-            int column_index = cursor.getColumnIndexOrThrow(MediaStore.Images.Media.DATA);
-            cursor.moveToFirst();
-            return cursor.getString(column_index);
-        } finally {
-            if (cursor != null) {
-                cursor.close();
-            }
-        }
-    }
-
 }
```

Comment:
Oh thanks! I thought I deleted it when I moved all the stuff to ScreenshotHelper.